playbooks/streisand.yml

```diff
@@ -20,7 +20,8 @@
       when: streisand_openvpn_enabled
     - role: shadowsocks
       when: streisand_shadowsocks_enabled
-    - ssh
+    - role: ssh-forward
+      when: streisand_ssh_forward_enabled
     - role: tinyproxy
       when: streisand_tinyproxy_enabled
```

Comment:
Actually: given the way we handled `streisand_stunnel_enabled` when openvpn is disabled I'm in favour of leaving the condition `when streisand_tinyproxy_enabled` and checking for `streisand_tinyproxy_enabled and not streisand_ssh_forward_enabled` in the validation tasks.